jhub_apps/config_utils.py

```diff
@@ -49,12 +109,25 @@ class JAppsConfig(SingletonConfigurable):
         help="The number of workers to create for the JHub Apps FastAPI service",
     ).tag(config=True)
 
-    allowed_frameworks = Bool(
+    allowed_frameworks = List(
```

Comment:
Flyby: seems like these should be Lists not Bools, but I could be wrong.

Reply:
Yes correct. Thanks for spotting this.